cypress/plugins/index.js

```diff
@@ -7,11 +9,29 @@
 // You can read more here:
 // https://on.cypress.io/plugins-guide
 // ***********************************************************
-
+const mockAPI = "http://localhost:3004/";
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 // eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on("task", {
```

Comment:
Oh we were okay with yall using axios here if it was working already in the task haha! Since you guys were already using axios before you can stick with that so you don't introduce another dependency. Sorry for not clarifying! That's just our approach since we never had axios in our project and we didn't want to mandate it